docs-v2/integrations/integration-templates/bamboohr-basic.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Bamboohr Basic API Integration Template'
+sidebarTitle: 'Bamboohr Basic'
```

Comment:
I know it's like that in providers but I still don't know: is it Bamboohr basic auth or "basic" plan or something like that?

Reply:
It is the Bamboohr basic auth.

Reply:
naming could be improved in the future :p